pillar/beacons.sls

```diff
@@ -2,6 +2,7 @@ beacons:
   network_settings:
     interval: 5
     # monitorize any change in IP addresses in eth0
+    # TODO: make this interface configurable
```

Comment:
Based on the documentation, couldn't we use `eth*` and other possible names for network cards? (https://docs.saltstack.com/en/latest/ref/beacons/all/salt.beacons.network_settings.html#salt.beacons.network_settings.beacon)

Reply:
We could just use `*`, but then any IP address change in any interface would lead to a `update-etc-hosts` orchestration.

The only solution I see is to implement our own beacon, but that would require some work...